packages/fx-core/src/plugins/resource/bot/azureOps.ts

```diff
@@ -137,9 +137,10 @@ export class AzureOperations {
     zipBuffer: Buffer,
     config: any
   ): Promise<string> {
+    const axiosInstance = axios.create();
```

Comment:
Could this be a static one instead of creating every deploy?

Reply:
Got it.